### ISSUE TITLE
Configure Project not working Apple Silicon

### ISSUE THREAD

**Issue by afreeland** [paulober/Pico-W-Go#80]
## What are the steps to reproduce this issue?
1. Install `Pico-W-Go`
2. Attempt to use `Pico-W-Go > Configure Project`
3. Status bar shows `Activating Extension` but nothing happens and there is no output of any kind, any where...

## What happens?
I am not sure that anything happens....

## What were you expecting to happen?
Initialize the project and board

## Any logs, error output, etc?
*(To get extension logs see Terminal > Output > Protcoll (Extensionhost) and copy output concerning picowgo)*
*(If it’s long, please paste to https://gist.github.com and insert the link here)*

```
2023-04-25 20:57:41.851 [info] ExtensionService#_doActivateExtension paulober.pico-w-go, startup: false, activationEvent: 'onCommand:picowgo.initialise'
```

## Any other comments?
I have limited...okay, zero knowledge on micropython stubs but I seem to be able to connect to the pico through cli

```
micropy stubs create /dev/cu.usbmodem2112401
```

![image](https://user-images.githubusercontent.com/3135030/234443954-dbc5899e-3765-4ca6-9161-0c83e783239a.png)

## Support info

Copy this from the `Help -> Info/About -> Copy` option in Visual Studio Code:

```
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:41:48.638Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: No```

**Comment by stig-bjorlykke** [comment]
Is this issue related to #65? I made it work on Apple M2 by installing Python 3.11.3 with [brew](https://brew.sh), not using Python 3.9.6 which is provided in macOS.

**Comment by PavelKalina** [comment]
Hi,
I have similar problém on Win 10 Enterprise.

Pico-W-Go v2.1.8 works without any problem.
No version 3.x worked. It whows "Activating Extension" for a while and then do nothing.
Version: 1.77.3 (user setup)
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:02.548Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: Yes

**Comment by couillonnade** [comment]
Same problem here on M1 macOS. Adding Python 3.11 instead of 3.10 solves the problem.

**Comment by PavelKalina** [comment]
Upgrade from Python 3.9 to 3.10 solved the problem.

**Comment by paulober** [comment]
@stig-bjorlykke @PavelKalina @couillonnade Thank you for all the feedback. The problem with Python 3.9 compatibility should be solved with Patch v3.0.8.
